src/SmartComponents/Actions/ListActions.js

```diff
@@ -4,7 +4,7 @@ import { Ansible, Battery, Main, PageHeader, PageHeaderTitle, RemediationButton,
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { Card, CardBody, CardHeader, Grid, GridItem, Stack, StackItem } from '@patternfly/react-core';
-import Breadcrumbs, { buildBreadcrumbs } from '../../PresentationalComponents/Breadcrumbs/Breadcrumbs';
+import Breadcrumbs from '../../PresentationalComponents/Breadcrumbs/Breadcrumbs';
```

Comment:
Is there any way we could use the breadcrumbs from patternfly instead?

Reply:
Nevermind. I see it.

Reply:
In the local Breadcrumbs component we're using PF4 components.  The local component just houses our logic.